dotCMS/src/main/java/com/dotmarketing/factories/TreeFactory.java

```diff
@@ -5,16 +5,15 @@
 import java.util.List;
 import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)